generated/.tailcallrc.schema.json

```diff
@@ -932,6 +966,94 @@
         }
       }
     },
+    "Resolver": {
```

Comment:
some new types was added to the schema, but they are not used actually. We can `serde(skip)` it after rewriting Resolver to proc macro